src/backend/templates/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-45266465-2"></script>
```

Comment:
Why load Analytics right at the beginning? Let's try to offer a better user experience by loading it last.

Reply:
I did this because google suggested to do so. Where would you suggest to put it, in the end of 'head'?